janikas-encinitas.md

```diff
@@ -0,0 +1,24 @@
+# Why Encinitas?
+---
+
+- [x] Beaches
+- [x] Sun
+- [x] Surf
+- [x] Mexican Food
+
+###Places to Eat
+* Rico's Taco Shop
+* Albertos
+* Robertos
+* Juanitas
+* Cardiff Market
+
+###Places to Go
```

Comment:
Used to... but I love the freedom of body surfing... just your fins and the waves